internal/stackdiag.go

```diff
@@ -394,6 +395,10 @@ func scheduleJobs(k *Kubectl, ns string, recordErr func(error), state *diagJobSt
 		}
 		tls := !(found && disabled)
 
+		if !filters.Contains(resourceName, typ) {
```

Comment:
I think you still need an empty check otherwise you disable stack diagnostics when no filters are defined.

Reply:
Thanks for seeing this.  Added this back.